assets/js/validation.js

```diff
@@ -16,6 +16,30 @@ const validateName = function () {
             validationResult.innerText = 'Error: ' + error;
         });
 };
-
 name.onkeyup = validateName;
 name.onchange = validateName;
+
+document.getElementById('komanda').addEventListener("click", komanda(), true);
+
+function komanda() {
+    let komanda = document.getElementById('komanda');
+    let validationResultTwo = document.getElementById('validation-result-two');
+
+    const validateTeam = function () {
+        validationResultTwo.innerText = "...";
+        axios.post(validationResultTwo.dataset.path, {input: name.value})
```

Comment:
Nalabai panašu, kad suvoki, kurie laukai, ką daro.
Nes dabar norime tikrinti `komandą`, bet siunti `name` reikšmę

```suggestion
        axios.post(validationResultTwo.dataset.path, {input: komanda.value})
```

**Vykdymas turėtų būti**:
  `komanda` keyup :arrow_right: `validateTeam` :arrow_right: `validationResultTwo.dataset.path` adresu su `komanda` laukelio reikšme